.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
 .nyc_output
 coverage/
-node_modules/
\ No newline at end of file
+node_modules/
+es.js
```

Comment:
any reason to ignore this file? it could be used even in GitHub pages to test some stuff live. Maybe `worker.js` or `web.js` would be a better name I could add in the exports dictionary too. What do you think?

Reply:
No reason, I was unsure how you liked to do this. I agree that `es.js` isn't the best name. Between those I think I like `worker.js` slightly better. Let me know what you want and I'll update it.